src/Raven.Client/Data/DatabaseStatistics.cs

```diff
@@ -79,6 +79,8 @@ public class IndexInformation
         public IndexType Type { get; set; }
 
         public DateTime? LastIndexingTime { get; set; }
+
+        public string OutputReduceToCollection { get; set; }
```

Comment:
We emit this only if it isn't null. This is an indication that the index is creating documents.